### PR TITLE
fix(TDI-44846):fix the compliation error.

### PR DESCRIPTION
**Pull request by vyu-talend**
**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TDI-44846

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.designer.components.localprovider/components/tSSH/tSSH_main.javajet [hunk 1]
```diff
@@ -42,7 +42,7 @@ if ((metadatas!=null)&&(metadatas.size()>0)) {
 			    log.info("<%=cid%> - Authentication using a password.");
 		    <%}%>
 		
-			session_<%=cid%> = client_<%=cid%>.connect(username_<%=cid%>, hostname_<%=cid%>, <%=("").equals(port)?"22":port%>).verify(<%=("true").equals(useTimeout)?timeout+"L  * 1000":""%>).getClientSession();
+			session_<%=cid%> = client_<%=cid%>.connect(username_<%=cid%>, hostname_<%=cid%>, Integer.parseInt(<%=port %>+"")).verify(<%=("true").equals(useTimeout)?timeout+"L  * 1000":""%>).getClientSession();
```
Comment (pyzhou-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tSSH/tSSH_main.javajet:45]: Although the user should not leave the port to empty, but we used to use the default 22 for empty port before my change. I think we should keep this behavior.

Comment (vyu-talend) [reply main/plugins/org.talend.designer.components.localprovider/components/tSSH/tSSH_main.javajet:45]: Hello, Pengyu. After testing with 7.3.1 which is before https://jira.talendforge.org/browse/TDI-42668,  I don't see the default value of the port in javajet, but I see the default value in XML file. And the exception is the same as before when the port is "" or null.

Comment (pyzhou-talend) [reply main/plugins/org.talend.designer.components.localprovider/components/tSSH/tSSH_main.javajet:45]: OK then

**Review by pyzhou-talend — APPROVED** [review]
LGTM

**Pull request merged**